### PR TITLE
Campaign landing page/ Shift Left

### LINKED ISSUE #5605: Shift left paid campaign page
Stakeholders: Nick and Fabiana
What?: Create a new page following the Shift left blog post. It won't be linked anywhere on the about platform, but we will include this on the Resources page once it's available.
Why?: Will be used in paid campaigns.

Assets:
🔗[Figma](https://www.figma.com/file/qGQBcYZ5NJCWI0Yejgfr7o/Shift-left?node-id=6%3A5914)
🔗[Meta](https://docs.google.com/spreadsheets/d/1DRUw6LgyNi-WXOQmx9JIZC6H_HUslvH2IegDJINCq6A/edit#gid=1477727999)

Asana: https://app.asana.com/0/1200095757619647/1202664491027174/f

### PR DESCRIPTION
**Pull request by st0nebraker**
Closes #5605. Adds new landing page for the Shift Left campaign

### Changelog
""

### Note
This will be included in the Resources page

### Test
1. Ensure prettier has standardized the proposed changes.
2. Nav to `/shift-left-the-good-parts` and ensure spec matches on all screen sizes

### CODE REVIEW

File: src/pages/shift-left-the-good-parts.tsx [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+                                Try Sourcegraph now
+                                <ArrowRightIcon className="ml-2" />
+                            </a>
+                        </Link>
+                    </div>
+                </section>
+            </ContentSection>
+        </div>
+
+        <ContentSection color="white" className="container max-w-900 py-7 px-0">
```
Comment (bretthayes) [inline src/pages/shift-left-the-good-parts.tsx:330]: The Figma doesn't show this having a smaller width constraint so I don't think we should add the max width. Was this a request?
```suggestion
        <ContentSection color="white" className="container py-7 px-0">
```

File: src/pages/shift-left-the-good-parts.tsx [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+            <div className="col-lg-6">
+                <h2 className="mb-5 font-weight-bold">Related resources</h2>
```
Comment (bretthayes) [inline src/pages/shift-left-the-good-parts.tsx:332]: Title in the Figma shows differently:
```suggestion
                <h2 className="mb-5 font-weight-bold">Read more about dev tools that accelerate the dev cycle</h2>
```

**Review by bretthayes — CHANGES_REQUESTED** [review]

File: src/pages/shift-left-the-good-parts.tsx [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+                                Consider cutting things that you might think are essential. Do you really need a
+                                production environment? Maybe use a tunneling tool like Ngrok and serve your product
+                                directly from your local machine.
+                            </li>
+                        </ul>
+                    </>
+                }
+                rightColumn={
+                    <Video
+                        host="self"
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/pages/shift-left-the-good-parts.tsx]: Ah! Good catch, it was rendering in development and something must've changed before I opened the PR. Got it going now 😎

**Comment by bretthayes** [comment]
Just something I noticed, but I think maybe we should add some top margin spacing to the caption under in the Video component @st0nebraker

**Comment by st0nebraker** [comment]
> Just something I noticed, but I think maybe we should add some top margin spacing to the caption in the Video component @st0nebraker

I think so too, especially with the video's getting a box shadow. I added a bit here, what do you think? @bretthayes 
image

**Comment by st0nebraker** [comment]
Also, what do you think about this pages slug? I think it should maybe have another content hierarchy folder, like `/campaign/` or something. It'll be included in the Resources page later, so I'm guessing we'll be using the content type: `campaign` or something similar. @bretthayes

**Comment by bretthayes** [comment]
> > Just something I noticed, but I think maybe we should add some top margin spacing to the caption in the Video component @st0nebraker
> 
> I think so too, especially with the video's getting a box shadow. I added a bit here, what do you think? @bretthayes image

That looks nice, and doesn't get hidden in the shadow. Thanks @st0nebraker!

**Comment by bretthayes** [comment]
> Also, what do you think about this pages slug? I think it should maybe have another content hierarchy folder, like `/campaign/` or something. It'll be included in the Resources page later, so I'm guessing we'll be using the content type: `campaign` or something similar. @bretthayes

I think this page falls under the same bucket as other pages that live under the root that are campaigned as well but it's not exposed to the user that it's a campaign via the URL structure. But maybe another name other than `campaign` would be a good idea for organizing these types of pages under a content type for the resources page. It would also help organize the file tree, so good idea! @st0nebraker

**Review by bretthayes — DISMISSED** [review]
This looks good to go! 🚀

**Comment by st0nebraker** [comment]
Small spacing & title change from stakeholder review. This is ready to ship! Just needs a ✅  @bretthayes

**Review by bretthayes — APPROVED** [review]

**Pull request merged**